### PR TITLE
Added ban duration SECOND setting on closeLobby

### PR DESCRIPTION
**Pull request by Rackover**
Untested yet

### CODE REVIEW

File: tests/unit_tests/test_lobbyconnection.py [hunk 1]
```diff
@@ -433,6 +471,41 @@ def test_send_mod_list(mocker, lobbyconnection, mock_games):
     assert len(bans) == 0
 
 
+async def test_command_admin_closelobby_with_ban_duration(mocker, lobbyconnection, db_engine):
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py]: Re-definition of the same test function?

Comment (Rackover) [reply tests/unit_tests/test_lobbyconnection.py]: (no ?)

File: tests/unit_tests/test_lobbyconnection.py [hunk 2]
```diff
@@ -398,6 +398,44 @@ def test_send_mod_list(mocker, lobbyconnection, mock_games):
     assert bans[0] == 'Unit test'
 
 
+async def test_command_admin_closelobby_with_ban_duration(mocker, lobbyconnection, db_engine):
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py]: Look here, it's the same function name

Comment (Rackover) [reply tests/unit_tests/test_lobbyconnection.py]: right! will fix

File: server/lobbyconnection.py [hunk 3]
```diff
@@ -296,6 +299,12 @@ def send_updated_achievements(self, updated_achievements):
                                         self._logger.warning('Tried to ban player with invalid period')
                                         raise ClientError(f"Period '{period}' is not allowed!")
 
+                                    self._logger.warning(">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")
```
Comment (Askaholic) [inline server/lobbyconnection.py]: This looks like left over stuff from when you were getting tests to work. Please delete

File: server/lobbyconnection.py [hunk 4]
```diff
@@ -283,6 +283,9 @@ def send_updated_achievements(self, updated_achievements):
                         reason = message['ban'].get('reason', 'Unspecified')
                         duration = int(message['ban'].get('duration', 1))
                         period = message['ban'].get('period', 'SECOND').upper()
+
```
Comment (Askaholic) [inline server/lobbyconnection.py]: Max 1 blank line please

File: tests/unit_tests/test_lobbyconnection.py [hunk 5]
```diff
@@ -15,7 +15,7 @@
 from server.players import Player, PlayerState
 from server.protocol import QDataStreamProtocol
 from server.types import Address
-from sqlalchemy import and_, select
+from sqlalchemy import and_, select, text, delete
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py]: `delete` is unused import. Also imports should be sorted

File: tests/unit_tests/test_lobbyconnection.py [hunk 6]
```diff
@@ -437,16 +437,22 @@ def test_send_mod_list(mocker, lobbyconnection, mock_games):
...
-            'command': 'admin',
-            'action': 'closelobby',
-            'user_id': banme.id,
-            'ban': {
-                'reason': 'Unit test',
-                'duration': 3600*24
-            }
-        })
+    """
+    Clearing database of previous unwanted bans
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py]: No point in making this a docstring, just make it a comment

File: server/lobbyconnection.py [hunk 7]
```diff
@@ -292,10 +295,16 @@ def send_updated_achievements(self, updated_achievements):
                                 if row:
                                     ban_fail = row[0]
                                 else:
-                                    if period not in ["DAY", "WEEK", "MONTH"]:
+                                    if period not in ["DAY", "WEEK", "MONTH", "SECOND"]:
```
Comment (Askaholic) [inline server/lobbyconnection.py]: Maybe we can sort these as well

File: server/lobbyconnection.py (hunk 3 above)
Comment (Rackover) [reply server/lobbyconnection.py]: 😅

File: tests/unit_tests/test_lobbyconnection.py [hunk 8]
```diff
@@ -425,6 +425,48 @@ def test_send_mod_list(mocker, lobbyconnection, mock_games):
...
+    player = mocker.patch.object(lobbyconnection, 'player')
+    player.login = 'Sheeo'
+    player.id = 1
+    player.admin = True
+    banme = mock.Mock()
+    banme.id = 200
+    lobbyconnection.player_service = {1: player, banme.id: banme}
+    lobbyconnection._authenticated = True
+
+    #Clearing database of previous unwanted bans
```
Comment (Askaholic) [inline tests/unit_tests/test_lobbyconnection.py]: Add space after `#`. https://www.python.org/dev/peps/pep-0008/#block-comments

**Pull request merged**